src/v/ssx/future-util.h

```diff
@@ -258,4 +260,55 @@ inline auto parallel_transform(Rng rng, Func func) {
       });
 }
 
+/// \brief Helper for explicitly ignoring a future's result, for use when
+/// backgrounding a task.
+///
+/// For example:
+///   ssx::background = ssx::spawn_with_gate_then...
+///
+/// This replaces use of (void) when spawning background fibers, and makes it
+/// easier to search for places we start background work.
+namespace detail {
+struct background_t {
+    template<typename T>
+    constexpr void operator=(T&&) const noexcept {}
+};
+} // namespace detail
+inline constexpr detail::background_t background;
```

Comment:
```suggestion
inline constexpr std::ignore_t background;
```
Checks docs "const /*unspecified*/ ignore"
```suggestion
inline constexpr auto background = ignore;
```
But I wonder if it can be expected that an unspecified can be copied?

Maybe just leave it as-is!

Reply:
Yeha, I think leaving it explicitly defined is the way to go, it doesn't seem like there are really any rules about what we can do with std::ignore other than assigning to it.